doc/7/controllers/security/get-user-strategies/snippets/get-user-strategies.js

```diff
@@ -0,0 +1,6 @@
+const response = await kuzzle.security.getUserStrategies('john.doe');
+
+console.log(response);
```

Comment:
Last nitpicking: better naming
```suggestion
const strategies = await kuzzle.security.getUserStrategies('john.doe');

console.log(strategies);
```